tasks/configure.yml

```diff
@@ -12,6 +12,7 @@
     - "/etc/grafana/provisioning/datasources"
     - "/etc/grafana/provisioning/dashboards"
     - "/etc/grafana/provisioning/notifiers"
+    - /etc/grafana/provisioning/plugins
```

Comment:
Could you surround it with `"` to be consistent with previous entries?

Reply:
Done.